content/tos/index.md

```diff
@@ -0,0 +1,113 @@
+
+1. Overview of the Service.
+   1. You may access and use the Service in accordance with this Agreement. You will comply with the terms of this Agreement and all laws, rules and regulations applicable to your use of the Service.
+   2. You may use any software, data, text, audio, video or images made available to you by any third party or in conjunction with the Service at your election (“Third-Party Content”). Third-Party Content is governed by this Agreement and, if applicable, separate terms and conditions accompanying such Third-Party Content, which terms and conditions may include separate fees and charges.
+   3. We reserve the right, at any time, to modify, suspend, or discontinue the Service, or any part thereof, upon thirty (30) days’ prior written notice to you, except that this notice will not be required if the thirty (30) day notice period (a) would pose a security or intellectual property issue to us or the Service, (b) is economically or technically burdensome, or (c) would cause us to violate legal requirements. You agree that Tlon will not be liable to you or to any third party for any modification, suspension, or discontinuance of the Service, or any part thereof.
+   4. This Agreement and the Service do not contain any service level obligations whatsoever with respect to the availability of the Service, whether by Tlon or otherwise. By entering into this Agreement, you acknowledge and agree that the Service may not be available to you at all times during the Term (defined below).
+2. Eligibility.
+   1. To access or use the Service, you must have a valid email address and a valid form of payment. You acknowledge and agree that you will only create one account per email address. During the user registration process for the Service, you will be required to agree to this Agreement.
+   2. During the registration process, you will be provided an Urbit ship if you do not already have one. You will be responsible for custody of the ownership keys of your ship. If you lose these, we cannot recover them for you, and you could lose access to your ship and your data permanently.
+   3. The Service is intended solely for individuals who are at least 18 years of age or older. Any registration on or use of the Service by any individual who does not meet the foregoing requirement is not authorized to register for or use the Service. By using the Service, you represent and warrant that you meet the foregoing requirement.
+3. Registration.
+   1. When you register for a user account, you will be asked to disclose certain personal information, including your name, your business address, the company or entity that you represent (if applicable), your billing information, your email address, and your selected password (“Account Information”), all of which will be subject to our Privacy Policy (available here, and provide us management credentials, but not ownership credentials, to your Urbit ship.
```

Comment:
> ... all of which will be subject to our Privacy Policy (available here, and provide us ...

I think this is missing a hyperlink and closing paren.

Reply:
Great catch. It is missing both of those.